src/app/app.component.scss

```diff
@@ -4,8 +4,19 @@
   height: 100%;
   overflow: hidden;
 
+  ccf-image-viewer-popover {
```

Comment:
Seems a little high up, why not in the ccf-image-viewer-popover?

Reply:
I moved all the styling into the image viewer CSS.